mbbsd/pfterm.c

```diff
@@ -28,6 +29,14 @@
 # endif
 #endif
 
+#ifndef FT_TRUE_ROWS
+# ifdef TERM_TRUE_ROWS
+#  define FT_TRUE_ROWS TERM_TRUE_ROWS // use the given true term height
```

Comment:
This is a violation of the separation of modules.

Instead, please try to rework the `resizeterm()` API, such that we pass in the "true" screen size _and_ the preferred screen size.

Reply:
Solution reworked.

I added a separate function `resizeterm_true()` to the API to keep the API backward-compatible.